test-result-summary-client/src/Build/ReleaseSummary.jsx

```diff
@@ -60,25 +60,33 @@ export default class ReleaseSummary extends Component {
                                 : ` ❌ ${buildResult} ❌${nl}`;
 
                         if (buildName.startsWith('Test_openjdk')) {
-                            const rerunLinkInfo = rerunLink
-                                ? `Rerun [all](${rerunLink}) ${nl}`
-                                : ``;
-
                             failedTestSummary[buildName] = buildInfo;
                             failedTestSummary[buildName] += buildResultStr;
 
                             if (!buildName.includes('_testList')) {
-                                failedTestSummary[buildName] += rerunLinkInfo;
+                                const rerunLinkInfo = rerunLink
+                                    ? `Rerun [all](${rerunLink})`
+                                    : ``;
```

Comment:
rerun link is not specific to _testList. Some test builds do not have child build, which means it does not have `_testList`. In short, the original logic is correct.

Reply:
So just move this logic outside the if statement?

Reply:
yes, that is correct